README.md

```diff
@@ -4,247 +4,148 @@ This backend is currently in development and is not advised for Production workf
+  - Supports core migration functionalities including creating, deleting, and updating indexes and collections
 
-`OPTIONS` is an optional dictionary of parameters that will be passed to
-[`MongoClient`](https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html).
 
-`USER`, `PASSWORD`, and `PORT` (if 27017) may also be optional.
+- **Index Management**  
+    
+  - Create single, compound, and unique indexes using Django Indexes  
+  - Create MongoDB partial indexes in Django using Q notation.
 
-For a replica set or sharded cluster where you have multiple hosts, include
-all of them in `HOST`, e.g.
-`"mongodb://mongos0.example.com:27017,mongos1.example.com:27017"`.
 
-Alternatively, if you prefer to simply paste in a MongoDB URI rather than parse
-it into the format above, you can use:
+- **Querying Data**  
+    
+  - Querying API powered through the amazing MongoDB Aggregation Pipeline  
+  - Supports most functions of the Django QuerySet API  
+  - Support Query Annotations and common SQL AGGREGATE operators  
+  - We support foreign keys and execute JOIN operations all in 1 database call  
+  - Through our custom raw\_aggregate call, MQL operations like Vector Search, Atlas Search, and GeoSpatial querying still yield Django QuerySet results\!
 
-```python
-import django_mongodb_backend
 
-MONGODB_URI = "mongodb+srv://my_user:my_password@cluster0.example.mongodb.net/myDatabase?retryWrites=true&w=majority&tls=false"
-DATABASES["default"] = django_mongodb_backend.parse_uri(MONGODB_URI)
-```
+- **Administrator Dashboard & Authentication**  
+    
+  - Manage your data in Django’s admin site.  
+  - Fully integrated with Django's authentication framework.  
+  - Supports native user management features like creating users and sessions.
 
-This constructs a `DATABASES` setting equivalent to the first example.
 
-#### `django_mongodb_backend.parse_uri(uri, conn_max_age=0, test=None)`
+- **Management Commands**  
+    
+  - Use commands like `migrate`, `makemigrations`, `flush`, `sqlmigrate`, many more.
 
-`parse_uri()` provides a few options to customize the resulting `DATABASES`
-setting, but for maximum flexibility, construct `DATABASES` manually as
-described above.
 
-- Use `conn_max_age` to configure [persistent database connections](
-  https://docs.djangoproject.com/en/stable/ref/databases/#persistent-database-connections).
-- Use `test` to provide a dictionary of [settings for test databases](
-  https://docs.djangoproject.com/en/stable/ref/settings/#test).
+## Future Commitments of Django Backend for MongoDB
 
-Congratulations, your project is ready to go!
+- **Advanced Indexing**  
+    
+  - Support for advanced index types like geospatial, text, and vector search indexes.  
+      
+- **Improved Data Modeling**  
+    
+  - Support ArrayFields containing Embedded Models  
+  - Support Collections with multiple Django Models  
```

Comment:
"collections" 

I don't think "mutiple" makes it clear that you mean polymorphic collections.